katdal/concatdata.py

```diff
@@ -206,18 +206,17 @@ class ConcatenatedSensorData(SensorData):
     This is a convenient container for returning raw (uncached) sensor data sets
     from a :class:`ConcatenatedSensorCache` object. It only accesses the
     underlying data sets when explicitly asked to via the __getitem__ interface,
-    but provides quick access to metadata such as sensor dtype, name and number
-    of data points.
+    but provides quick access to metadata such as sensor dtype and name.
 
     Parameters
     ----------
-    data : sequence of recarray-like with fields ('timestamp', 'value', 'status')
-        Uncached sensor data as a list of record arrays or equivalent (such as a
-        :class:`h5py.Dataset`)
+    data : sequence of recarray-like with fields 'timestamp', 'value' ('status')
```

Comment:
That wasn't completely clear from the comment. Maybe "'timestamp', 'value', and optionally 'status'"